src/components/LoanCards/ListLoanCard.vue

```diff
@@ -53,15 +53,15 @@
 						@favorite-toggled="toggleFavorite"
 					/>
 				</div>
-				<div class="small-7 medium-8 large-5 xlarge-6 xxlarge-7 columns">
+				<div class="small-7 medium-8 large-5 xlarge-6 columns">
 					<borrower-info-header
 						:country="loan.geocode.country.name"
 						:name="loan.name"
 						:loan-id="loan.id"
 						class="list-loan-card-borrower-info-header"
 					/>
 				</div>
-				<div class="large-7 xlarge-6 xxlarge-5 columns show-for-large">
+				<div class="large-7 xlarge-6 columns show-for-large">
```

Comment:
I had to adjust these since the card was smaller at the same viewport size in this experiment